site/src/pages/docs/client-factory.mdx

```diff
@@ -34,6 +36,28 @@ You can also use <type://ClientFactoryOptions> to build <type://ClientFactory>.
 It's more verbose than calling a dedicated setter method, but it can be useful when you need to
 set a series of options programmatically.
 
+## Customizing TLS
+
+You can configure the TLS key and certificates
+for [mutual TLS (mTLS)](https://www.cloudflare.com/ko-kr/learning/access-management/what-is-mutual-tls/)
+using <type://ClientFactoryBuilder#tls(InputStream,InputStream)> .
+Additionally, you can specify various options such as a different CA certificate chain or enforce certain 
+cipher suites with <type://ClientFactoryBuilder#tlsCustomizer(Consumer)> .
+
+```java
+String resourceRoot = "/path/to/resource/";
+ClientFactory builder =
+  ClientFactory
+    .builder()
+    .tls(getClass().getResourceAsStream(resourceRoot + "custom.crt"),
```

Comment:
Won't this leak resources? try-with-resource isn't as pretty and makes the code less clear, but this seems to imply that the `.tls()` is closing the streams (which the `SslContextBuilder` api says it does not)

Reply:
Nice catch. It should be closed.